Library will not compile if LV_DRAW_SW_COMPLEX is disabled

### LVGL version

v9.2.0

### What happened?

When configuring the library to use the simple renderer by setting `LV_DRAW_SW_COMPLEX` to 0 in `lv_conf.h`, the library would not compile with the following error:
```bash
In file included from ../Drivers/lvgl/src/tick/../core/lv_global.h:49,
                 from ../Drivers/lvgl/src/tick/lv_tick.c:11:
../Drivers/lvgl/src/tick/../core/../draw/sw/lv_draw_sw_mask_private.h:38:5: error: unknown type name 'lv_draw_sw_mask_xcb_t'
```

The type `lv_draw_sw_mask_xcb_t` is defined in the file `lv_draw_sw_mask.h` but under the macro condition `#if LV_DRAW_SW_COMPLEX`.

So the library won't compile unless `LV_DRAW_SW_COMPLEX` is defined as `#define LV_DRAW_SW_COMPLEX 1`.

### How to reproduce?

To reproduce, just set the following option in `lv_config.h`
```c
#define LV_DRAW_SW_COMPLEX    0
```
and build.

--- a/src/draw/sw/lv_draw_sw_mask_private.h
+++ b/src/draw/sw/lv_draw_sw_mask_private.h
@@ -16,6 +16,8 @@ extern "C" {
 
 #include "lv_draw_sw_mask.h"
 
+#if LV_DRAW_SW_COMPLEX
+
 /*********************
  *      DEFINES
  *********************/
@@ -145,6 +147,8 @@ void lv_draw_sw_mask_cleanup(void);
 /**********************
  *      MACROS
  **********************/
+
+#endif  /*LV_DRAW_SW_COMPLEX*/
 
 #ifdef __cplusplus
 } /*extern "C"*/

